src/ape/managers/project/dependency.py

```diff
@@ -192,12 +192,6 @@ class GithubDependency(DependencyAPI):
     such as ``dapphub/erc20``.
     """
 
-    # TODO: Remove at >= 0.7
-    branch: Optional[str] = None
```

Comment:
Make sure we communicate this change well

Reply:
I updated the PR notes to better reflect this, but will do an even better job when it comes to publish 0.7